sources/zarr/large_image_source_zarr/__init__.py

```diff
@@ -1179,7 +1179,7 @@ def _generateDownsampledLevels(self, resample_method):
                     output=iterator_output,
                     resample=False,  # TODO: incorporate resampling in core
                 ):
-                    new_tile = downsampleTileHalfRes(tile['tile'], resample_method)
+                    new_tile = downsampleTileHalfRes(np.squeeze(tile['tile']), resample_method)
```

Comment:
We only want to remove the last component if it has length 1.  Otherwise, something like this will fail:
```
sink = large_image.new()
sink.addTile(np.zeros((1, 1024, 1), dtype=np.uint8))
sink.write('sample.db')
```